source/css/_layout.grids.scss

```diff
@@ -253,17 +253,24 @@
   }
 }
 
+/**
+ * Grid 7up
+ */
 .l-grid--7-col {
   width: 100%;
 
   > * {
-    width: calc(100%/7);
+    padding-left: $pad;
+    padding-right: $pad;
+
+    @include media('>medium') {
+      width: calc(100vw/7);
+    }
   }
 
   &--l--offset-1 {
     @include media('>large') {
-      margin-left: calc(100%/7);
-      width: calc(100% - (100% / 7));
+      margin-left: calc(100vw/7);
```

Comment:
We should add spaces between the values and operators in these `calc()` functions.